akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/ActorContext.scala

```diff
@@ -93,6 +93,9 @@ trait ActorContext[T] { this: akka.actor.typed.javadsl.ActorContext[T] ⇒
    * given [[ActorRef]] terminates. This message is also sent when the watched actor
    * is on a node that has been removed from the cluster when using akka-cluster
    * or has been marked unreachable when using akka-remote directly
+   *
+   * Will fail with an IllegalStateException if the same subject was watched before using `watchWith`.
```

Comment:
`[[IllegalStateException]]`? (Not that people does not know what it is though, but always nice with navigability)

Reply:
:+1: